### ISSUE TITLE
`execaSync()` does not work but `await execa()` does

### ISSUE THREAD

**Issue by felipecrs** [denoland/deno#20782]
https://github.com/sindresorhus/execa is a great tool that simplifies command executions in Node. I'm trying to convert my applications to use Deno and I stumbled upon this issue.

```ts
// test.ts

import {execaSync, execa} from 'npm:execa@8.0.1';

let result = await execa('echo', ['unicorns async']);
console.log(result.stdout);

result = execaSync('echo', ['unicorns sync']);
console.log(result.stdout);
```

```console
❯ deno run -A test.ts
unicorns async
error: Uncaught (in promise) Error: Command failed: echo unicorns sync
    at makeError (file:///home/felipecrs/.cache/deno/npm/registry.npmjs.org/execa/8.0.1/lib/error.js:60:11)
    at execaSync (file:///home/felipecrs/.cache/deno/npm/registry.npmjs.org/execa/8.0.1/index.js:201:17)
    at file:///tmp/tmp.Qq2z9kWPKd/test.ts:6:10
    at Object.runMicrotasks (ext:core/01_core.js:848:30)
    at processTicksAndRejections (ext:deno_node/_next_tick.ts:53:10)
    at runNextTicks (ext:deno_node/_next_tick.ts:71:3)
    at eventLoopTick (ext:core/01_core.js:189:21)
```

**Comment by felipecrs** [comment]
Just to update: the same problem still happens as of deno 1.38.0.

**Comment by bartlomieju** [comment]
This should be fixed in Deno v1.38.1 later this week.
